static/js/stream_ui_updates.js

```diff
@@ -140,6 +141,20 @@ export function update_change_stream_privacy_settings(sub) {
     }
 }
 
+export function update_stream_privacy_icon_in_settings(sub) {
+    $(
```

Comment:
We can return early here if the edit panel is not opened like we do for other similar update functions using `hash_util.is_editing_stream`.